plugins/bundle/config.go

```diff
@@ -172,8 +173,20 @@ func (c *Config) validateAndInjectDefaults(services []string, keys map[string]*k
 			}
 		}
 
+		// If the resource specifies a file:// URL then we can ignore the
+		// service configuration error.
+		ignoreServiceConfigErr := false
+
+		if strings.HasPrefix(source.Resource, "file://") {
+			if u, err := url.Parse(source.Resource); err != nil || u.Scheme != "file" {
+				return fmt.Errorf("invalid URL for bundle %q: %v", name, err)
```

Comment:
Something is odd here, isn't it? We first check that the prefix is `file://`, and then check the `u.Scheme` again? I've been wondering because if `err` is ni and `u.Scheme` is **not** `"file"`, the error would look a bit funny:
```
invalid URL for bundle xyz: nil
```
...but I cannot come up with a `source.Resource` string that would trigger that. 🤔

Reply:
Agreed. The scheme will always be file because of the prefix check. Removed the second condtion.